cmd/zed/dev/indexfile/create/command.go

```diff
@@ -59,6 +60,8 @@ func newCommand(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
 	})
 	f.StringVar(&c.outputFile, "o", "index.zng", "name of index output file")
 	f.StringVar(&c.keys, "k", "", "comma-separated list of field names for keys")
+	f.IntVar(&c.opts.ZNGWriterOpts.LZ4BlockSize, "znglz4blocksize", zngio.DefaultLZ4BlockSize,
+		"LZ4 block size in bytes for ZNG compression (nonpositive to disable)")
```

Comment:
The behavior of `-znglz4blocksize=0` is different here than it is everywhere else. I think you should make this consistent, but if you keep it as is, fix the usage string.
```suggestion
		"LZ4 block size in bytes for ZNG compression (negative to disable)")
```

Reply:
> The behavior of -znglz4blocksize=0 is different here than it is everywhere else. I think you should make this consistent

I don't quite follow, I literally copied this from outputflags.

Reply:
A couple of other things:

Setting LZ4BlockSize to 0 disables compressions, (so the negative to disable comment is not right):
```
func NewWriter(w io.WriteCloser, opts WriterOpts) *Writer {
	var comp *compressor
	if opts.LZ4BlockSize > 0 {
		comp = &compressor{}
	}
	return &Writer{
		writer:     w,
		compressor: comp,
		opts:       opts,
		types:      NewEncoder(),
		thresh:     opts.LZ4BlockSize, //XXX
	}
}
```
Additionally any value <= 0 for WriterOpts.LZ4BlockSize makes the Writer output a new value block for every value:
```
func (w *Writer) Write(val *zed.Value) error {
...
	if len(w.values) >= w.thresh || len(w.types.bytes) >= w.thresh {
		return w.flush()
	}
```
Which makes the current output for LZ4BlockSize < 0 pretty unusable (unless have a bunch of single value zed blocks is not a big deal).

Reply:
> I don't quite follow, I literally copied this from outputflags.

I know, but in `index.NewWriterWithContext` you're adding
```go
	if opts.ZNGWriterOpts.LZ4BlockSize == 0 {
		opts.ZNGWriterOpts.LZ4BlockSize = zngio.DefaultLZ4BlockSize
	}
```
and that means `zed dev indexfile create -znglz4blocksize=0` gives you compression with the default block size, rigfht?

Reply:
Oh shoot I forgot I put that in there. You are right this is weird. Since I already merged this, I'll put up a separate pr to fix this.